raiden-ts/src/channels/reducer.ts

```diff
@@ -198,12 +159,12 @@ const channels = createReducer(initialState.channels)
   .handle(channelOpen.request, channelOpenRequestReducer)
```

Comment:
Please, can you explicitly define the types of `createReducer`s to e.g. `Reducer<RaidenState['channels'], RaidenAction>` (on all `createReducer` usage, adapting the `S` type)? I've noticed the intersection type can get quite heavy on tsc/tsserver, and is part of the slowdown you saw. I'm looking for a proper fix, but this workaround is compatible and good enough for now (it is a `Reducer` afterall).

Reply:
Sure :+1: